bugsnag-plugin-android-ndk/src/main/jni/utils/serializer/json_writer.c

```diff
@@ -275,6 +275,15 @@ void bsg_serialize_stackframe(bugsnag_stackframe *stackframe, bool is_pc,
     json_object_set_string(frame, "method", (*stackframe).method);
   }
 
+  if (*stackframe->code_identifier != 0) {
+    char code_identifier[sizeof(stackframe->code_identifier) + 1];
+    code_identifier[sizeof(stackframe->code_identifier)] =
+        0; // force zero terminator
```

Comment:
What is the expected length for these? Are they consistent?

Reply:
They're not strictly consistent and depend on which hash option is being used: https://android.googlesource.com/toolchain/llvm-project/+/refs/heads/master/lld/ELF/SyntheticSections.cpp#278